webhook/README.rst

```diff
@@ -0,0 +1,143 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======
+Webhook
+=======
+
+Module to receive .. _global webhooks: https://en.wikipedia.org/wiki/Webhook events.
```

Comment:
This part does not render well on the module page (check the runbot), you could rewrite it this way:
```
Module to receive `webhook <https://en.wikipedia.org/wiki/Webhook/>`_ events.
```